### PR TITLE
Add net.krafting.SemantiK

### PR DESCRIPTION
**Pull request by Krafting**
### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### App Description

SemantiK is a french word game where you need to find a secret word, similar to cémantix (or semantle). It also have a sharing feature where we can share the current word to find with friends. 

You can also use your own word_lists and language models.

### CODE REVIEW

**Comment by Krafting** [comment]
bot, build net.krafting.SemantiK

File: net.krafting.SemantiK.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  - name: OpenBLAS
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+    - -DBUILD_SHARED_LIBS=ON
+    - -DBUILD_TESTING=OFF
+    sources:
+      - type: git
+        url: https://github.com/OpenMathLib/OpenBLAS
+        branch: v0.3.26
```
Comment (hfiguiere) [inline net.krafting.SemantiK.yml:23]: don't use branch. tag + commit is the recommended way

File: net.krafting.SemantiK.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DBUILD_SHARED_LIBS=ON
+    - -DBUILD_TESTING=OFF
+    - -DCMAKE_Fortran_COMPILER=gfortran
+    - -DLAPACKE=ON
+    - -DCBLAS=ON
+    sources:
+      - type: git
+        url: https://github.com/Reference-LAPACK/lapack.git
+        branch: v3.12.0
```
Comment (hfiguiere) [inline net.krafting.SemantiK.yml:38]: don't use branch. tag + commit is the recommended way

File: net.krafting.SemantiK.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      - type: git
+        url: https://github.com/mesonbuild/meson-python
+        tag: 0.15.0
+
+  - name: SciPy
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/scipy/scipy.git
+        branch: v1.12.0
```
Comment (hfiguiere) [inline net.krafting.SemantiK.yml:60]: don't use branch. tag + commit is the recommended way

File: net.krafting.SemantiK.yml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  - python3-pythran.json
+
+  - python3-pybind11.json
+
+  - name: meson-python
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/mesonbuild/meson-python
+        tag: 0.15.0
```
Comment (hfiguiere) [inline net.krafting.SemantiK.yml:53]: why this when it is in  `python3-meson-python.json`?

File: net.krafting.SemantiK.yml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  - name: OpenBLAS
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+    - -DBUILD_SHARED_LIBS=ON
+    - -DBUILD_TESTING=OFF
+    sources:
+      - type: git
+        url: https://github.com/OpenMathLib/OpenBLAS
+        tag: v0.3.26
```
Comment (hfiguiere) [inline net.krafting.SemantiK.yml:23]: tag + commit

File: net.krafting.SemantiK.yml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      - type: git
+        url: https://github.com/mesonbuild/meson-python
+        tag: 0.15.0
+
+  - name: SciPy
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/scipy/scipy.git
+        tag: v1.12.0
```
Comment (hfiguiere) [inline net.krafting.SemantiK.yml:60]: tag + commit

File: net.krafting.SemantiK.yml [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+      - install -Dm 0644 net.krafting.SemantiK.desktop  /app/share/applications/net.krafting.SemantiK.desktop
+      - install -Dm 0644 net.krafting.SemantiK.metainfo.xml  /app/share/metainfo/net.krafting.SemantiK.metainfo.xml
+      - install -Dm 0644 images/net.krafting.SemantiK.png /app/share/icons/hicolor/256x256/apps/net.krafting.SemantiK.png
+    sources:
+      - type: file
+        url: https://embeddings.net/embeddings/frWac_non_lem_no_postag_no_phrase_200_cbow_cut100.bin
+        sha256: d51f3259d1f6fe12155a447623a82695c6d72916b1b61e150e2715c034b6c880
+      - type: git
+        url: https://gitlab.com/Krafting/semantik-gtk
+        tag: 1.0
```
Comment (hfiguiere) [inline net.krafting.SemantiK.yml:79]: tag + commit

**Comment by Krafting** [comment]
bot, build net.krafting.SemantiK

**Comment by Krafting** [comment]
bot, build net.krafting.SemantiK

**Comment by Krafting** [comment]
bot, build net.krafting.SemantiK

**Comment by Krafting** [comment]
bot, build net.krafting.SemantiK

**Pull request closed without merging**